extensions/bitwarden/src/api.ts

```diff
@@ -0,0 +1,38 @@
+import execa from "execa";
+
+export class BitwardenApi {
+  private env: Record<string, string>;
+  constructor(clientId: string, clientSecret: string) {
+    this.env = {
+      BW_CLIENTSECRET: clientSecret,
+      BW_CLIENTID: clientId,
+    };
+  }
+
+  async syncItems(sessionToken: string): Promise<void> {
+      await this.exec(["sync", "--session", sessionToken])
```

Comment:
Since this command passes a `sessionToken` to the `bw` command, does it need to be called with `this.exec()`, which includes the API key details?